### PR TITLE
List

### PR DESCRIPTION
**Pull request by JaanviNL**

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+@import url(/main.css);
+@import url(/components/alert/alert.css);
+
+.unordered-list
+{
+    list-style: none;
+    border-top:0.1rem solid black;
+    border-left: 0.1rem solid black;
+    border-right: 0.1rem solid black;
```
Comment (Aditi0505) [inline components/list/list.css]: Can create a variable for color here and can use hex color.

File: components/list/list.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    width: 20rem;
+    margin: auto;
+}
+.list-item
+{
+    padding: 0.5rem 1rem;
+    border-bottom: 0.1rem solid black;;
```
Comment (Aditi0505) [inline components/list/list.css]: Can create a variable for color here and can use hex color. Can remove extra ; here.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good to me!
You can also provide description and screenshot for better visibility.

**Review by aurobindoGupta — APPROVED** [review]
add screenshot.
nice work

**Pull request merged**